File: src/main/java/net/eduard/permissions/command/PermissionsCommand.java

package net.eduard.permissions.command;

import net.eduard.api.lib.manager.CommandManager;

public class PermissionsCommand extends CommandManager {

	public PermissionsCommand() {
		super("permissions");
		
		register(new PermissionsGroupCommand());
		register(new PermissionsUserCommand());
		register(new PermissionsHelpCommand());
		register(new PermissionsSaveCommand());
		register(new PermissionsReloadCommand());
	}
	

}

File: src/main/java/net/eduard/permissions/command/PermissionsCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command

class PermissionsCommand : Command("permissions","perm","permission") {
    init {
        register(PermissionsGroupCommand())
        register(PermissionsUserCommand())
        register(PermissionsHelpCommand())
        register(PermissionsSaveCommand())
        register(PermissionsReloadCommand())
    }
}